install/scripts/alpaca.sh

```diff
@@ -8,11 +8,5 @@ if [ -f "$HOME_DIR/islandora/install/configs/variables" ]; then
   . "$HOME_DIR"/islandora/install/configs/variables
 fi
 
-cd "$HOME_DIR"
-git clone https://github.com/Islandora-CLAW/Alpaca.git
-cd Alpaca
```

Comment:
If the install we provide with the source code doesn't actually work on the source code it's shipped with, then why is it even here?  I say we pull it out to another repo at that point.

If we must dodge the submodules, then in the very least can we push the git remote and branch to variables?  I'm using the submodules to avoid having to constantly edit my build scripts when testing installs of code that hasn't been accepted yet.